### DISCUSSION TITLE
Is there any way to change the time format from 24h to 12h?

### DISCUSSION THREAD

**Discussion by alexislayvu** [LazyVim/LazyVim#2309] [Q&A] [answered]
<img width="371" alt="Screenshot 2024-01-04 at 7 46 35 PM" src="https://github.com/LazyVim/LazyVim/assets/54639928/f674f603-3cc9-407d-a9a6-245dba3be7e1">

**Comment by dpetka2001** [comment, marked as answer]
```lua
-- ~/.config/nvim/lua/plugins/notify.lua
return {
  {
    "rcarriga/nvim-notify",
    opts = {
      time_formats = {
        notification = "%I:%M:%S",
      },
    },
  },
}
```
should do it I believe. Test it to see if it's what you want.

**Reply by alexislayvu** [reply]
that did the trick, thank you so much!!!
